src-admin-routes/admin-routes.ts

```diff
@@ -1,11 +1,17 @@
 import express from 'express';
-import { deleteProverb, editProverb, approveProverb, getProverbs } from './admin-controllers';
+import { deleteProverb, editProverb, approveProverb, getProverbs, getUsers, searchUsers } from './admin-controllers';
+import { paginateResponse } from '../services/paginateResponse'
+import Proverb from '../models/proverb.js'
+import User from '../models/user.js'
+
 import checkAdmin from './checkAdmin'
 const router = express.Router();
 
 router.use(checkAdmin)
 
-router.get('/all-proverbs', getProverbs);
+router.get('/all-proverbs', paginateResponse(Proverb), getProverbs);
+router.get('/search', searchUsers);
```

Comment:
is this returning all the users?? If the answer is yes, I would use a url following the REST API standars. like:

/users/?search="hello"